docs/features/multiple-networks/multi-network-policies.md

```diff
@@ -1,30 +1,56 @@
 # MultiNetworkPolicies
-OVN-Kubernetes implements native support for
-[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
-an API providing
+
+## Introduction
+
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy) provides
 [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
-features for secondary networks.
+features for secondary networks, and allow enhanced traffic security.
+
+## Motivation
+
+Using traffic restrictions based on user network configuration is a recommended practice. By defining policies that
+specify which services a workload can access, we mitigate supply chain attacks. Even if a component is compromised, it
+won’t be able to jeopardize other components of the system.
```

Comment:
can you point to any document recommending this practice ? 

I would point to the kubernetes networking model definition where it is explained that network policies are used to block traffic, and by default everything is reachable. 

This is the same concept but adapted to the secondary networks.

I'm not so keen on having the particular example of the supply chain attach here. Maybe it can be here, but please highlight it is just an example. When I read this, it seems the only thing about multi-net policies is preventing supply chain attacks.